Doc/whatsnew/3.12.rst

```diff
@@ -98,6 +98,61 @@ Deprecated
 ==========
 
 
+Pending Removal in Python 3.13
+==============================
+
+The following APIs have been deprecated in earlier Python releases,
+and will be removed in Python 3.13.
+
+Removed modules:
```

Comment:
each module mentions PEP 594, maybe just mention it here?

Reply:
Yep, updated!